concent_api/concent_api/system_check.py

```diff
@@ -174,6 +174,55 @@ def check_settings_storage_cluster_address(app_configs, **kwargs):  # pylint: di
     return []
 
 
+@register()
```

Comment:
please add unit tests for new system checks

Reply:
:+1: